test/app_templates/config/initializers/shopify_app_with_webhooks.rb

```diff
@@ -1,4 +1,5 @@
 ShopifyApp.configure do |config|
+  config.application_name = "name"
```

Comment:
You don't need to add the name everywhere. It's an optional parameter. Only add it to the `shopify_app.rb` file that you're using to test its presence in the config.